coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java

```diff
@@ -320,6 +320,10 @@ public boolean isOptional(int i) {
     // Date Functions
     createAddUserDefinedFunction("from_unixtime", FunctionReturnTypes.STRING,
         family(ImmutableList.of(SqlTypeFamily.NUMERIC, SqlTypeFamily.STRING), optionalOrd(1)));
+    createAddUserDefinedFunction("trino_from_unixtime", explicit(SqlTypeName.TIMESTAMP),
```

Comment:
Is it possible to create an operator (e.g., `TimestampFromUnixtime`) and have its unparse method return `to_unixtimestamp` and add it to the HiveFunctionRegistry instead of sticking to 
the standard way of creating operators through `createAddUserDefinedFunction`? I think this can save us from having to create `TrinoFromUnixtimeOperatorTransformer`?